force_wfmanager/left_side_pane/workflow_tree.py

```diff
@@ -200,8 +236,12 @@ def _workflow_mv_default(self):
 
     @on_trait_change('model')
```

Comment:
If there is a way to decorate an on_trait_change with @verify_wkflow, it would be nice as at the minute verify_workflow_event has to be triggered within the function